<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapMainWindow</class>
 <widget class="QMainWindow" name="MapMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="MapCreateView" name="graphicsView"/>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="lookRoomBT">
          <property name="text">
           <string>look检视房间</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="saveRoomBT">
          <property name="text">
           <string>保存房间并刷新连线</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="cmdLE"/>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QSplitter" name="splitter_3">
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="midLineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="RoomWidget" name="mapWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>206</width>
              <height>438</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_2">
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="outRoomWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>205</width>
               <height>216</height>
              </rect>
             </property>
            </widget>
           </widget>
           <widget class="QScrollArea" name="scrollArea_3">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="inRoomWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>205</width>
               <height>215</height>
              </rect>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="openMap"/>
    <addaction name="saveMap"/>
   </widget>
   <widget class="QMenu" name="menuDLL">
    <property name="title">
     <string>DLL</string>
    </property>
    <addaction name="adddll"/>
    <addaction name="deleteDLL"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>其它</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuDLL"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openMap">
   <property name="text">
    <string>打开地图</string>
   </property>
  </action>
  <action name="saveMap">
   <property name="text">
    <string>保存地图</string>
   </property>
  </action>
  <action name="adddll">
   <property name="text">
    <string>加载DLL</string>
   </property>
  </action>
  <action name="deleteDLL">
   <property name="text">
    <string>卸载DLL</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapCreateView</class>
   <extends>QGraphicsView</extends>
   <header>mapCreateWindow/mapcreateview.h</header>
  </customwidget>
  <customwidget>
   <class>RoomWidget</class>
   <extends>QWidget</extends>
   <header>showWidget/roomwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
